documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+# Getting Started: how to process data for Timelapse Feature Explorer
```

Comment:
Markdown preview here: https://github.com/allen-cell-animated/colorizer-data/blob/doc/getting-started-guide/documentation/GETTING_STARTED.md

Reply:
Also note that some of the links are broken since they point to where date *will be* once this is merged. I'll fix this when I have time so they work for now and then revert the change before it goes to main.